Guide and minor main page changes for 1.1.0 (#609)
mongodb/terraform-provider-mongodbatlas: website/docs/index.html.markdown

## Configure Atlas Programmatic Access

> In order to set up authentication with the MongoDB Atlas provider a programmatic API key must be generated for MongoDB Atlas with the appropriate [role](https://docs.atlas.mongodb.com/reference/user-roles/) and IP access list entry.

Reviewer: remove passive voice
```suggestion
In order to set up authentication with the MongoDB Atlas provider, you must generate a programmatic API key for MongoDB Atlas with the appropriate [role](https://docs.atlas.mongodb.com/reference/user-roles/) and IP access list entries.
```

The [MongoDB Atlas documentation](https://docs.atlas.mongodb.com/tutorial/manage-programmatic-access/index.html) contains the most up-to-date instructions for creating and managing your key(s), setting the appropriate role, and IP access.  

> Role: If unsure which role level to grant your key we suggest creating an Organization API Key with an Organization Owner role as that will ensure you have sufficient access for all actions.

Reviewer: minor edits for clarity
```suggestion
**Role**: If unsure of which role level to grant your key, we suggest creating an organization API Key with an Organization Owner role. This ensures that you have sufficient access for all actions.
```

> IP access list: Not all API resources require an IP access list by default but some do, such as backup, hence we highly suggest adding an IP access list as soon as possible.  See [Require IP Access List for Public API](https://docs.atlas.mongodb.com/tutorial/manage-organization-settings/#require-ip-access-list-for-public-api) for more info.

Reviewer: minor edits for clarity
```suggestion
**IP access list**: Some API resources, such as backup resources, require an IP access list by default. We highly suggest that you add an IP access list as soon as possible.  See [Require IP Access List for Public API](https://docs.atlas.mongodb.com/tutorial/manage-organization-settings/#require-ip-access-list-for-public-api) for more info.
```
